tests/utils/profilehandler/profile_handler.go

```diff
@@ -240,7 +240,13 @@ func GenerateClusterCreateFlags(profile *Profile, client *rosacli.Client) ([]str
 			flags = append(flags,
 				"--audit-log-arn", auditRoleArn)
 		}
+
+		if profile.ClusterConfig.AdditionalPrincipals {
```

Comment:
I remember the additional-allowed-principals should come from another AWS account

Reply:
@xueli181114 Should not it be covered in e2e api test only, how can we use different aws account in rosa?